kedro/framework/context/context.py

```diff
@@ -335,8 +335,7 @@ def _get_config_credentials(self) -> Dict[str, Any]:
         """Getter for credentials specified in credentials directory."""
         try:
             conf_creds = self.config_loader["credentials"]
-        except MissingConfigException as exc:
-            warn(f"Credentials not found in your Kedro project config.\n{str(exc)}")
```

Comment:
Could we simply do this at debug level instead of fully removing it?

Reply:
Wouldn't that still be misleading to users who are parsing through their credentials without using the credentials.yml file?

Reply:
I don't think so, we don't mention anything about `credentials.yml` file, we just say there's no config for the credentials, which could mean that there's no file, or fetching the credentials failed in any other way.